docs/csharp/language-reference/keywords/using-statement.md

```diff
@@ -16,6 +16,10 @@ The following example shows how to use the `using` statement.
 
 [!code-csharp[csrefKeywordsNamespace#4](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsNamespace/CS/csrefKeywordsNamespace.cs#4)]
 
+Beginning with C# 8.0, you can use the following alternative syntax for the `using` statement:
```

Comment:
would it be helpful to extend this to say something like "that doesn't require braces" or something to this extent?